### ISSUE TITLE
Logged-out view does not hide some labeled images

### ISSUE THREAD

**Issue by vyv03354** [bluesky-social/social-app#3265]
**Describe the bug**
Logged-out view does not hide some labeled images even if it should.

**To Reproduce**

1. Open [this skeet](https://bsky.app/profile/nousk.jp/post/3knywn6isfk2x) in private window. (Warning: the image is labeled as sexually suggestive.)

**Expected behavior**

The image should be hidden.

**Screenshots**

Omitted due to the nature of the problem.

**Details**

 - Platform: web
 - Platform version: Firefox 123.0.1 on Windows 11 23H2 build 22631.3296
 - App version: Build version: 1.73.0; Platform: web
(I heard older versions worked as expected.)

**Additional context**

The image is hidden correctly if you logged in to bsky.
[The API response](https://public.api.bsky.app/xrpc/app.bsky.feed.getPostThread?uri=at%3A%2F%2Fdid%3Aplc%3Ab64z7ls556ptckcuup3wbpl6%2Fapp.bsky.feed.post%2F3knywn6isfk2x) contains the label, so the app seems to fail to apply the label for some reason.

**Comment by surfdude29** [comment]
Confirmed, I can see the image when logged out.

**Comment by surfdude29** [comment]
This seems to also apply to images labelled Pornography, but only those labelled by the Bluesky Moderation Service, not those labelled by the author.

** ***Warning: All of these linked images are definitely NSFW*** **

**Visible when logged out, labelled by Bluesky Moderation Service:**

https://bsky.app/profile/pnwtwunkwannabe.bsky.social/post/3knz7r23sla2d

https://bsky.app/profile/gaypornboi1.bsky.social/post/3ko2l2rpksd2e

https://bsky.app/profile/phdrs.bsky.social/post/3knzxck3p6v2w


**Not visible when logged out, labelled by author:**

https://bsky.app/profile/userradu.bsky.social/post/3knz22d7td22a

https://bsky.app/profile/newfiemunky.bsky.social/post/3knyysyexkt2r

https://bsky.app/profile/happywanksgivin.bsky.social/post/3knyj46xn3d2e

**Comment by pfrazee** [comment]
Thanks for the heads up!

**Comment by nouskjp** [comment]
Nice to meet you. I am a bluesky user from Japan. I came here because I have the same problem. Thanks for the clear report. Like me, many of my followers are NSFW artists and were concerned about this problem. I hope it will be improved.

The following is in Japanese, but here is the URL I asked my follower if the sensitive label does not work after logout.
[](https://bsky.app/profile/nousk.jp/post/3knz6zjzanx2j)
I have seen from two of my other followers that the filter does not show up when logging out.

I do not understand English. This text was translated by deepL. I hope there is no mistranslation.

**Comment by nouskjp** [comment]
Nice to meet you. I am a bluesky user from Japan. I came here because I have the same problem. Thanks for the clear report. Like me, many of my followers are NSFW artists and were concerned about this problem. I hope it will be improved.

The following is in Japanese, but here is the URL I asked my follower if the sensitive label does not work after logout.

https://bsky.app/profile/nousk.jp/post/3knz6zjzanx2j

I have seen from two of my other followers that the filter does not show up when logging out.

I do not understand English. This text was translated by deepL. I hope there is no mistranslation.

**Comment by nouskjp** [comment]
I am a Blue Sky user from Japan. I commented because I have the same problem as this bug report.
Thank you for your detailed report. Like me, many of my followers are NSFW artists and were concerned about this issue. I hope it will be improved.

Here is the URL I asked my followers if the sensitive label does not work after logout, although it is in Japanese.
https://bsky.app/profile/nousk.jp/post/3knz6zjzanx2j

My environment
macmini macos13 
chrome version: 122.0.6261.129
safari version: 18615.3.12.11.2
iphone12 ios17.1.1 chrome safari

I saw from two other followers that the filter does not show up when they log out.
Follower's browser
Brave
Google browser for tablets

I do not understand English. This text was translated by deepL. I hope there is no mistranslation.
